### PR TITLE
Fix #2122

### LINKED ISSUE #2122: Displaying the list of routes linked to a document grouped by activity
To prevent many scrolling, Displaying the list of routes linked to a document grouped by activity with the possibility of opening or closing one activity

### PR DESCRIPTION
**Pull request by jeanphilippeds**
Hello,

Here is a PR to close #2122:
Capture d’écran 2022-09-28 à 11 48 09

### CODE REVIEW

File: src/views/document/utils/boxes/RoutesBox.vue [hunk 1]
```diff
@@ -27,11 +29,14 @@
 </template>
 
 <script>
+import AccordionItem from '@/components/generics/AccordionItem.vue';
 import { requireDocumentProperty } from '@/js/properties-mixins';
 
 export default {
+  components: {
```
Comment (brunobesson) [inline src/views/document/utils/boxes/RoutesBox.vue:36]: Thanks to https://github.com/c2corg/c2c_ui/blob/master/src/js/vue-plugins/generic-components.js, you do not need to declare the components from `generics` folder

File: src/views/document/utils/field-viewers/MarkdownSection.vue [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+    </accordion-item>
   </div>
 </template>
 
 <script>
+import AccordionItem from '@/components/generics/AccordionItem.vue';
 import { requireDocumentProperty, requireFieldProperty } from '@/js/properties-mixins';
 
 export default {
+  components: {
```
Comment (brunobesson) [inline src/views/document/utils/field-viewers/MarkdownSection.vue:26]: Thanks to https://github.com/c2corg/c2c_ui/blob/master/src/js/vue-plugins/generic-components.js, you do not need to declare the components from `generics` folder

**Review by brunobesson — APPROVED** [review]
Nice one, thanks!

Only one minor comment.

**Comment by AntoineMarnat** [comment]
Nice ! Thank you

**Pull request merged**